resources.go

```diff
@@ -2209,6 +2209,55 @@ func Provider() tfbridge.ProviderInfo {
 				"Pulumi":                       "1.5.0-*",
 				"System.Collections.Immutable": "1.6.0",
 			},
+			Namespaces: map[string]string{
+				"appsync":           "AppSync",
+				"appmesh":           "AppMesh",
+				"apigateway":        "ApiGateway",
+				"appautoscaling":    "AppAutoScaling",
+				"autoscaling":       "AutoScaling",
+				"cloudformation":    "CloudFormation",
+				"cloudhsmv2":        "CloudHsmV2",
+				"cloudfront":        "CloudFront",
+				"cloudtrail":        "CloudTrail",
+				"cloudwatch":        "CloudWatch",
+				"codebuild":         "CodeBuild",
+				"codecommit":        "CodeCommit",
+				"codedeploy":        "CodeDeploy",
+				"codepipeline":      "CodePipeline",
+				"datapipeline":      "DataPipeline",
+				"datasync":          "DataSync",
+				"devicefarm":        "DeviceFarm",
+				"directoryservice":  "DirectoryService",
+				"docdb":             "DocDB",
+				"dynamodb":          "DynamoDB",
+				"directconnect":     "DirectConnect",
+				"ec2clientvpn":      "Ec2ClientVpn",
+				"ec2transitgateway": "Ec2TransitGateway",
+				"elasticache":       "ElastiCache",
+				"elasticbeanstalk":  "ElasticBeanstalk",
+				"elasticsearch":     "ElasticSearch",
+				"elastictranscoder": "ElasticTranscoder",
+				"lb":                "LB",
+				"gamelift":          "GameLift",
+				"globalaccelerator": "GlobalAccelerator",
+				"guardduty":         "GuardDuty",
+				"licensemanager":    "LicenseManager",
+				"lightsail":         "LightSail",
+				"mediapackage":      "MediaPackage",
+				"mediastore":        "MediaStore",
+				"opsworks":          "OpsWorks",
+				"redshift":          "RedShift",
+				"resourcegroups":    "ResourceGroups",
+				"securityhub":       "SecurityHub",
+				"secretsmanager":    "SecretsManager",
+				"servicecatalog":    "ServiceCatalog",
+				"servicediscovery":  "ServiceDiscovery",
+				"servicequotas":     "ServiceQuotas",
+				"simpledb":          "SimpleDB",
+				"storagegateway":    "StorageGateway",
+				"wafregional":       "WafRegional",
+				"worklink":          "WorkLink",
```

Comment:
these are the list of names that were clearly wrong.  but there are open questions about names like `emr`.  Should we make that ElasticMapReduce?  Or should we make it Emr.  The latter corresponds to JS/Python.  The former is more .NET-y.

Reply:
Probably the best thing to do here would be to follow the AWS .net sdk's for consistency with them.  What do you think @lukehoban @pgavlin @mikhail